gnd/src/main/java/com/google/android/gnd/ui/offlinebasemap/viewer/OfflineBaseMapViewerFragment.java

```diff
@@ -109,6 +112,11 @@ public void onRemoveClick() {
       return;
     }
 
-    viewModel.onRemoveClick();
+    Timber.d("Removing offline area %s", viewModel.getOfflineArea());
+    viewModel
+        .onRemoveClick()
+        .doOnComplete(() -> navigator.navigateUp())
```

Comment:
Please move this logic into subscribe() instead of side effect.

Reply:
done!